src/Util/Test.php

```diff
@@ -190,10 +191,14 @@ public static function getRequirements($className, $methodName)
         $requires = [];
 
         if ($count = \preg_match_all(self::REGEX_REQUIRES_OS, $docComment, $matches)) {
-            $requires['OS'] = \sprintf(
-                '/%s/i',
-                \addcslashes($matches['value'][$count - 1], '/')
-            );
+            foreach (\range(0, $count - 1) as $i) {
+                $requires[$matches['name'][$i]] = $matches['name'][$i] === 'OSFAMILY'
+                    ? $matches['value'][$i]
+                    : \sprintf(
+                        '/%s/i',
+                        \addcslashes($matches['value'][$i], '/')
```

Comment:
logic when to use strictly provided value and when to decorate it for regex matching can be moved direct to place, when we compare the `'OS'` value. I'm for that movement, but with that, we would need to update all related tests (to not contain `/Linux/i` but just `Linux`). WDYT @sebastianbergmann ?